src/admin/package.json

```diff
@@ -24,6 +24,7 @@
     "babel-loader": "8.1.0",
     "babel-plugin-transform-export-extensions": "^6.22.0",
     "bootstrap": "^4.6.0",
+    "he": "^1.2.0",
```

Comment:
This package is very old and doesn't appear to be well maintained.  A quick search found https://github.com/mdevils/html-entities which is a lot more active and may be a better choice.

Reply:
Thanks @marklise. I swapped it to html-entities and it seems to behave exactly the same as he.